Add Antora docs with the Decidim documentation format (#99)
decidim/decidim-bulletin-board: docs/modules/configure/pages/environment_variables.adoc

= Environment Variables

> We try to let most of the things defined through Environment Variables, as https://12factor.net/config[12factor applications's config]. You'll need to configure through your hosting provider (for example in Heroku), through a gem like `figaro` or through docker-compose `env_file`.

Reviewer: ```suggestion
We try to let most of the things defined through Environment Variables, as https://12factor.net/config[12factor applications's config]. You'll need to configure them your hosting provider (for example in Heroku), through a gem like `figaro` or through docker-compose `env_file`.
```

Reviewer 2: I guess `them` was meant to be added after the word `through`, rather replace it? @slickepinne

Reviewer: Yeah. 👍

.Decidim Bulletin Board Env Vars
|===
|Name |Value

|CORS_ORIGIN_ALLOWED
|The domains allowed to access to the API. Use `*` for public Bulletin Boards.

|DATABASE_URL
|Connection string for the Postgres database

|IDENTIFICATION_PRIVATE_KEY
|The Bulletin Board private key, in JWT format. It should at least include the keys `kty`, `n`, `e`, `d`, `p`, `q`, `dp`, `dq` and `qi`.

|RAILS_ENV

> |Environment for Ruby on Rails. If it's on your local machine for development, then it should be `development`. If it's on server it should be `production`. Please take care that this has security implications. You DON'T want to be running a production application on development mode.

Reviewer: ```suggestion
|Environment for Ruby on Rails. If it's on your local machine for development, then it should be `development`. If it's on server it should be `production`. Please take care that this has security implications. You DON'T want to run a production application in development mode.
```

|RAILS_LOG_TO_STDOUT

> |If true it'll show the log contents on the shell Standard Out. It's special relevant on certain hosting providers (for example, Heroku or Docker). It's also part of https://12factor.net/logs[12factor's recomendations].

Reviewer: ```suggestion
|If true it will show the log contents on the shell Standard Out. It's especially relevant for certain hosting providers (for example, Heroku or Docker). It's also part of https://12factor.net/logs[12factor's recommendations].
```

|SECRET_KEY_BASE

> |Secret key base for the Application. It's specially important that this is kept secret from the outside word; do NOT publish it on GitHub/GitLab/BitBucket.

Reviewer 2: Though that `specially` was misspelled, but TIL that both `specially` and `especially` exist, and they have a very similar meaning: https://www.vocabulary.com/articles/chooseyourwords/especially-specially/#:~:text=Specially%20means%20%22for%20a%20special,and%20a%20shamrock%20or%20two.
I guess either will fit here 👍

|SEED

> |Used to force seed test data on production environments.

Reviewer 3: ```suggestion
|Used to force seed test data on production environments, if `true`.
```
